molvis/molvisbase/include/inviwo/molvisbase/util/atomicelement.h

```diff
@@ -224,6 +230,11 @@ constexpr double covalentRadius(Element symbol) noexcept {
 constexpr double atomicMass(Element symbol) noexcept {
     return detail::atomic_mass[atomicNumber(symbol)];
 }
+constexpr bool isNonMetallic(Element symbol) noexcept {
```

Comment:
Why not just have isMetallic and put the implementation there?

Reply:
As the list of elements includes only non-metallic ones, this seemed more reasonable.